fetch.bs

```diff
@@ -7800,11 +7819,16 @@ method steps are:
  <li><p>Return <var>p</var>.
 </ol>
 
-<p>To <dfn>abort fetch</dfn> with a <var>promise</var>, <var>request</var>, and
-<var>responseObject</var>, run these steps:
+<p>To <dfn>abort fetch</dfn> with a <var>promise</var>, <var>request</var>,
```

Comment:
It would help a lot to rename this something like "abort a fetch() call". I keep getting it confused with "abort" as used on fetch controllers, which is more general; this is about messing with the JavaScript objects specifically vended by fetch().

Reply:
Renamed :)